src/mlpack/methods/ann/ffn.hpp

```diff
@@ -142,6 +142,16 @@ class FFN
   void Predict(arma::mat& predictors, arma::mat& results);
 
   /**
+   * Predict the responses to a given set of predictors. The responses will
+   * reflect the output of the given output layer as returned by the
+   * output layer function.
+   *
+   * @param predictors Input predictors.
+   * @return Output predictions of responses into.
+   */
+  arma::mat Predict(arma::mat predictors);
+
```

Comment:
Sounds good for me, I think we should use a reference here, there might be compilers which are not going to optimize this way.